responses/test_responses.py

```diff
@@ -891,13 +893,17 @@ def test_response_filebody():
     """ Adds the possibility to use actual (binary) files as responses """
 
     def run():
+        current_file = os.path.abspath(__file__)
         with responses.RequestsMock() as m:
-            with open("README.rst", "r") as out:
+            with open(current_file, "r") as out:
                 m.add(responses.GET, "http://example.com", body=out.read(), stream=True)
                 resp = requests.get("http://example.com")
-            with open("README.rst", "r") as out:
+            with open(current_file, "r") as out:
                 assert resp.text == out.read()
 
+    run()
```

Comment:
@markstory 
for some reason in this test was no call to the run()
I added call to enable this test, if it was intentional, then let me know and I will remove it back

Reply:
@markstory 
can you please trigger test run?